geomesa-gs-styling/src/main/scala/org/geomesa/gs/styling/filter/GeoMesaLabelParser.scala

```diff
@@ -0,0 +1,41 @@
+/***********************************************************************
+ * Copyright (c) 2013-2017 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the GNU GENERAL PUBLIC LICENSE,
+ * Version 2 which accompanies this distribution and is available at
+ * https://opensource.org/licenses/GPL-2.0.
+ ***********************************************************************/
+
+package org.geomesa.gs.styling.filter
+
+import java.lang.{Double => jDouble, Object => jObject}
+
+import org.geotools.filter.FunctionExpressionImpl
+import org.geotools.filter.capability.FunctionNameImpl
+import org.geotools.filter.capability.FunctionNameImpl._
+
+class GeoMesaLabelParser extends FunctionExpressionImpl(
+  new FunctionNameImpl("geomesaParseLabel",
```

Comment:
That's a good point but I'm not sure given it's in the geomesa-geoserver project.